repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -358,10 +364,9 @@ mount -o "remount,rw" "$NEWROOT"
     # check if leapp previously failed in the initramfs, if it did return to the emergency shell
     [ -f "$NEWROOT$LEAPP_FAILED_FLAG_FILE" ] && {
         echo >&2 "Found file $NEWROOT$LEAPP_FAILED_FLAG_FILE"
-        echo >&2 "Error: Leapp previously failed and cannot continue, returning back to emergency shell"
+        echo >&2 "Warning: Leapp failed on a previous execution and something might be blocking the upgrade."
+        echo >&2 "If possible, try to boot to the original system or restore the system from a backup."
```

Comment:
I would add info that any other following error could be potentially misleading as there was a problem discovered before which could affect any other leapp execution.

Reply:
@pirat89 yes, I am still discussing the error message with CEE right now. Should have put WIP there. Sorry